src/banman.cpp

```diff
@@ -165,18 +166,19 @@ bool BanMan::Unban(const CSubNet& sub_net)
 
 void BanMan::GetBanned(banmap_t& banmap)
 {
-    LOCK(m_cs_banned);
+    LOCK(m_banned_mutex);
     // Sweep the banlist so expired bans are not returned
     SweepBanned();
     banmap = m_banned; //create a thread safe copy
 }
 
 void BanMan::SweepBanned()
 {
+    AssertLockHeld(m_banned_mutex);
+
```

Comment:
nit: This extra line can be removed.

```suggestion
```

Reply:
Of course, it's only a personal opinion, but for it is not an `extra` line. It is a line which separates assertions from the function logic that helps to make the presence of assertions clearer.

Reply:
I agree with you. However, I think it is better to follow a code uniformity. And some places have a line between, and some don't. This breaks the code uniformity. But these are just my 2 cents, and I am okay with whatever you think would be the optimal choice in this matter.